src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -625,11 +633,12 @@ void CouplingSchemeConfiguration::addAbsoluteConvergenceMeasure(
                     << "limit=\"" << limit << "\" "
                     << "data=\"" << dataName << "\" "
                     << "mesh=\"" << meshName << "\" "
-                    << "suffices=\"" << suffices << "\"/> subtag in your <coupling-scheme ... /> in the precice-config.xml.");
+                    << "/> subtag in your <coupling-scheme ... /> in the preCICE configuration file.");
   impl::PtrConvergenceMeasure measure(new impl::AbsoluteConvergenceMeasure(limit));
   ConvergenceMeasureDefintion convMeasureDef;
   convMeasureDef.data        = getData(dataName, meshName);
   convMeasureDef.suffices    = suffices;
+  convMeasureDef.strict      = strict;
```

Comment:
Does this require special treatment?

Doesn't this mean that all other convergence measures are superfluous.

Reply:
You mean when you set both strict and suffices to true? Yes, exactly. But this could be helpful for users if they not want to comment out all other convergence measures. And, you could of course mark two measures as strict. Then, "strict" overrules "suffices". Of course, all of that are edge cases, but they should be covered in a meaningful way.

Reply:
Ok, so there is a use case.